pkg/cmd/render/render_test.go

```diff
@@ -218,7 +218,7 @@ func TestRenderCommand(t *testing.T) {
 		testFunction  func(cfg *kubecontrolplanev1.KubeAPIServerConfig) error
 	}{
 		{
-			name: "scenario 1 checks feature gates",
+			name: "checks feature gates",
```

Comment:
For my info, why is it desirable to remove scenario numbering? This seems like an unrelated non-functional change.

Reply:
You are getting the name of the test in a fail report anyway, so you have something to look up the test by. If you remove a test somewhere in the middle, or it feels only right to add a test in the middle of the array of the tests, you would have to to renumber. I do not see the value of test numbers.